earth2studio/models/dx/cbottle_sr.py

```diff
@@ -403,126 +384,149 @@ def load_model(
-            grid. If false, the native nested HealPix grid will be used for input and output.
-            Input HEALPix is level 6 and output HEALPix is level 10 with NEST pixel ordering.
+            grid. If false, the native nested HEALPix grid will be used for input and output
+            Input HEALPix is level 6 and output HEALPix is level 10 with NEST pixel ordering
         output_resolution : tuple[int, int], optional
             High-resolution output dimensions for lat/lon output, by default (2161, 4320)
-        super_resolution_window : None | tuple[int, int, int, int], optional
+        super_resolution_window : tuple[int, int, int, int] | None, optional
             Super-resolution window for lat/lon output, by default None
         sampler_steps : int, optional
             Number of diffusion steps, by default 18
         sigma_max : float, optional
             Noise amplitude used to generate latent variables, by default 800
+        distilled_model : bool, optional
+            Whether to use the distilled model, by default False. If True, the distilled helper is used,
+            enabling generation with fewer sampler steps
+        window_function : str, optional
+            Window smoothing function to apply when combining multidiffusion patches
+            Only supported when distilled_model is True
+        window_alpha : float, optional
+            Window parameter (e.g. alpha for KBD windows)
+            Only supported when distilled_model is True
         seed : int, optional
             Random generator seed for latent variables, by default None
+        device : str | torch.device, optional
+            Device on which to instantiate the underlying cBottle model, by default CUDA when available
 
         Returns
         -------
         DiagnosticModel
             Diagnostic model
         """
+        if device is None:
+            device = "cuda:0" if torch.cuda.is_available() else "cpu"
+        torch_device = cls._as_torch_device(device)
+
+        checkpoint_name = (
+            "cBottle-SR-Distill.zip" if distilled_model else "cBottle-SR.zip"
+        )
+        model_cls = (
+            DistilledSuperResolutionModel if distilled_model else SuperResolutionModel
+        )
 
-        with Checkpoint(package.resolve("cBottle-SR.zip")) as checkpoint:
-            core_model = checkpoint.read_model()
+        state_path = package.resolve(checkpoint_name)
+
+        load_kwargs = {
+            "num_steps": sampler_steps,
+            "sigma_max": sigma_max,
+            "device": str(torch_device),
+        }
+
+        if distilled_model:
+            if window_function is None:
+                window_function = "uniform"
+            if window_alpha is None:
+                window_alpha = 1.0
+            load_kwargs["window_function"] = window_function
+            load_kwargs["window_alpha"] = window_alpha
+        else:
+            if window_function is not None or window_alpha is not None:
```

Comment:
Why would you want to run without the window function if distilled? Should these just get turned on if people are using the distilled model?